### DISCUSSION TITLE
Comparision with other ML inference frameworks?

### DISCUSSION THREAD

**Discussion by lainisourgod** [sonos/tract#482] [Ideas]
Hi 👋!

I'm very, **very** excited about adopting this framework for my project's ML inference, but I'm struggling comparing it in-depth with different solutions (myriads of them, 😵).

As such, it would be cool to know how do you compare tract to others (torch jit, tf lite, onnxruntime, ncnn, tvm, .......). Do you do some periodic evaluation or maybe gaining influence from what other frameworks do?

I guess such comparsion would be beneficial for framework adoption.

**Comment by kali** [comment]
Well, we have very scarse resources, we prefer to put them into making tract better than running comparison with framework we are not experts of. We are tracking tract performance very seriously, but not comparing it too much with other frameworks. We are targetting a semi-niche in the ecosystem.

You are very welcome to try to run benches or comparisons. We will help you or anybody by answering questions about tract and give your tips about how to make the most of it.

**Reply by lainisourgod** [reply]
Thanks! I hope experience with tract will be great for my use case.
